src/core/tags.js

```diff
@@ -193,7 +193,7 @@ Object.assign(Tags.prototype, {
      * @function
      * @name pc.Tags#add
      * @description Add a tag, duplicates are ignored. Can be array or comma separated arguments for multiple tags.
-     * @param {string} name - Name of a tag, or array of tags.
+     * @param {string|string[]} name - Name of a tag, or array of tags.
```

Comment:
does the processor all a "rest" argument, so we can capture the 3rd "any number of strings"?  Would `{string|string[]|...string}` be legal?

Reply:
Gosh, this is a tricky one. I can add `...string` which generates `...name: string[]`. But doing `...string|string[]` outputs `name: string[]` which is not what we want. So I've heavily relaxed the types to have `...*` which does work for all possible params (although of course does allow for other types like booleans and numbers say. If you have a better approach, let me know, otherwise, we can just merge this.